srv/components/misc_pkgs/kafka/start.sls

```diff
@@ -70,7 +82,7 @@ Ensure kafka has started:
         # The reverse happens when shell we have a true condition with ret-code '0'. until results in false.
         # Note this when interpreting the next line.
         until: True
-        attempts: 10
-        interval: 2
+        attempts: 12
+        interval: 10
```

Comment:
Keep the interval to small, I think 5 should be good. But increase the retry count to large value like 30. This was we will be responsive yet wait for long time.

Reply:
@ujjwalpl , @sachinpunadikar 
These suggestions were implemented:
https://github.com/Seagate/cortx-prvsnr/pull/1317/files#diff-f2e1b47d101c4aa5f1493a7361ccb300dbbd4e4bfc7d9c510a219cd83a92e59cR85

Kindly reconsider.